eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/GA4GHBeaconResponse.java

```diff
@@ -0,0 +1,83 @@
+package uk.ac.ebi.eva.server.ws.ga4gh;
+
+public class GA4GHBeaconResponse {
+
+    private String chromosome;
+
+    private Integer start;
+
+    private String allele;
+
+    private String datasetIds;
+
+    private boolean exists;
+
+    private String errorMessage;
+
+    public GA4GHBeaconResponse() {
+    }
+
+    public GA4GHBeaconResponse(String chromosome, Integer start, String allele, String datasetIds, boolean exists) {
+        this.chromosome = chromosome;
+        this.start = start;
+        this.allele = allele;
+        this.datasetIds = datasetIds;
+        this.exists = exists;
+    }
+
+    public GA4GHBeaconResponse(String chromosome, Integer start, String allele, String datasetIds, String errorMessage) {
+        this.chromosome = chromosome;
+        this.start = start;
+        this.allele = allele;
+        this.datasetIds = datasetIds;
+        this.errorMessage = errorMessage;
+    }
+
+    public String getChromosome() {
```

Comment:
Boilerplate. Tested in file tracking. The only precaution one has to do is avoiding bidirectional class relationships. This is fixed by using a specific annotation that jackson needs to identify and handle it himself.